<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TogetherClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/login/images/loginpage/back.svg);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(255, 255, 255, 0.1);
border-radius: 40px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="accountLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 21px;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="connexionLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font-family: 'Gilroy';
font-style: normal;
font-weight: 700;
font-size: 38px;
line-height: 46px;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passWordInput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #BCBEC0;
border-radius: 10px;
color:black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>yourPassWord</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="accountInput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #BCBEC0;
border-radius: 10px;
color:black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="placeholderText">
      <string>Potter78</string>
     </property>
    </widget>
    <widget class="QLabel" name="passWordLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 21px;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>PassWord</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BD0C47;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="forgotPassWordLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 12px;
line-height: 14px;
color: #FFFFFF;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Forgot pass word</string>
     </property>
    </widget>
    <widget class="QLabel" name="continueWithLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>or continue with</string>
     </property>
    </widget>
    <widget class="QPushButton" name="googleButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #BCBEC0;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/login/images/loginpage/google.svg</normaloff>:/login/images/loginpage/google.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="gitButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #BCBEC0;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/login/images/loginpage/git.svg</normaloff>:/login/images/loginpage/git.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="facebookButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>280</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #BCBEC0;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/login/images/loginpage/fb.svg</normaloff>:/login/images/loginpage/fb.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../rcs/qtrcs.qrc"/>
 </resources>
 <connections/>
</ui>
